### DISCUSSION TITLE
Toast generator playground missing imports

### DISCUSSION THREAD

**Discussion by mgrtomasjurik** [boris-jenicek/ngx-awesome-popup#23] [Bug report]
Hello,
thx for this great tool - it's really helpful and easy to use 😀👍

I've found a bug on the playground toast generator v 2.2.0
https://costlydeveloper.github.io/ngx-awesome-popup/#/playground/toast-generator

The imports are missing the new animations: AppearanceAnimation, DisappearanceAnimation

The complete list should be:
import {
  ToastNotificationInitializer,
  DialogLayoutDisplay,
  ToastUserViewTypeEnum,
  ToastProgressBarEnum,
  ToastPositionEnum,
  DisappearanceAnimation,
  AppearanceAnimation,
} from '@costlydeveloper/ngx-awesome-popup';

**Comment by boris-jenicek** [comment]
Ok thank you, i will check that soon.

**Comment by boris-jenicek** [comment]
demo site updated.

**Reply by mgrtomasjurik** [reply]
Nice, thx 😀👍
